apps/src/templates/teacherHomepage/UiTip.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const UiTip = React.createClass({
+  propTypes: {
+    index: React.PropTypes.number,
+    position: React.PropTypes.object,
+    text: React.PropTypes.string,
+    closeClicked: React.PropTypes.func.isRequired
+  },
+
+  closeClicked(index) {
+    this.props.closeClicked(index);
+  },
+
+  render() {
+    const { index, position, text } = this.props;
+
+    return (
+      <div>
+        <div
+          className="arrow_box_up"
```

Comment:
Do we need to use an after selector vs. just sticking a styled DOM element after this one? If this answer is that we do need to use :after, I would advocated for still having the non-after stuff inlined (but dont feel super strongly).